src/style/app-fixed.module.css

```diff
@@ -4020,30 +4020,160 @@
 
 /* -- GeneralSettings.tsx -- */
 
+.cardSettings {
+  border-radius: 0.5rem;
+  margin-bottom: 1.5rem;
+  border: none;
+  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.075);
+  height: 800px;
+  width: 588px;
+}
+
+.cardHeaderStyle {
+  border-top-left-radius: 0.25rem;
+  border-top-right-radius: 0.25rem;
+  padding: 1.25rem 1rem 1rem 1rem;
+  border-bottom: 3px solid var(--cardHeader-border);
+  background-color: #eaebef;
+  padding: 16px;
+}
+
+.userupdatediv {
+  padding: 25px 16px;
+  background: var(--white-color);
+}
+
+.maskGroupButton {
+  width: 20px;
+  height: 20px;
+  color: #1778f2;
+}
+
+.customTextarea {
+  min-height: 100px;
+  background-color: #f1f3f6;
+  color: #737373;
+}
+
+.textFields {
+  background-color: #f1f3f6;
+  color: #737373;
```

Comment:
Use a variable that matches this element and function. There are similar examples at the top of the file